trans.py

```diff
@@ -19,8 +19,11 @@
 
 import os
 
-os.system('crowdin-cli-py upload sources')
-os.system('crowdin-cli-py download')
+home_dir = os.path.expanduser('~')
+user_config = os.path.join(home_dir, 'crowdin.yaml')
```

Comment:
never mind.. https://github.com/crowdin/crowdin-cli#split-project-configuration-and-user-credentials :) `crowdin.yaml` it is..